curp/src/server/raw_curp/mod.rs

```diff
@@ -893,7 +900,10 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
             lst_r.term
         };
 
-        let next_index = self.lst.get_next_index(follower_id);
+        let Some(next_index) = self.lst.get_next_index(follower_id) else {
+            warn!("follower {} is not found, it maybe has been removed", follower_id);
```

Comment:
If the follower has been removed, please make sure that the leader stop all the sync task.

Reply:
all the sync task? Do you mean the sync task of the removed node? the task will be stop when this method return `None`